exercises/isogram/.meta/generator/isogram_case.rb

```diff
@@ -6,9 +6,19 @@ def workload
     indent_lines(
       [
         "string = #{input.inspect}",
-        "#{assert} Isogram.is_isogram?(string)"
+        "#{assert} Isogram.isogram?(string), #{failure_message}"
       ], 4
     )
   end
 
+  private
+
+  def failure_message
+    %Q("Expected '#{expected}', '#{input}' #{type}")
+  end
+
+  def type
+    "#{expected ? 'is' : 'is not'} an isogram"
```

Comment:
Comment on @kotp's edit: `input` is the input - the string that is provided as input to the method under test.